Misc/NEWS.d/next/Core and Builtins/2019-05-02-11-48-08.bpo-36774.ZqbJ1J.rst

```diff
@@ -0,0 +1,4 @@
+Add a ``!x`` conversion specifier to f-strings. This is similar to
```

Comment:
Is it `!d`? I couldn't see references for `!x` in the PR.

Reply:
Yes, it's `!d`. Thanks for catching this. I started off using `!x` and missed this one. I'll fix it.